src/js/index.js

```diff
@@ -1,52 +1,78 @@
-const title = document.getElementById('title');
-const author = document.getElementById('author');
-const btn = document.getElementById('btn');
-const storageList = document.getElementById('storage_list');
-if (!localStorage.getItem('count')) {
-  localStorage.setItem('count', 0);
-}
-if (!localStorage.getItem('books')) {
-  localStorage.setItem('books', JSON.stringify([]));
+class Ls {
```

Comment:
- [ ] **The newly added book disappears when the page is refreshed. Please fix this.**

When I add a book, the data is persisted in the local storage. Great job with that. However, the book disappears from the page when I refresh it. Please fix it. 

![a](https://user-images.githubusercontent.com/59168430/124647903-2ddefc80-deb4-11eb-93a3-3bceb4619316.gif)